lnbits/static/js/wallet.js

```diff
@@ -223,6 +223,12 @@ new Vue({
             align: 'right',
             label: 'Fiat Amount',
             field: row => row.extra.wallet_fiat_amount
+          },
+          {
+            name: 'items',
+            align: 'right',
+            label: 'Items',
+            field: row => (row.tag === 'tpos' && row.extra.details) || null
```

Comment:
This is layering violation but I guess I agree with the quick fix like this.

Maybe there should be a way how an extension can tell LNbits which extra fields should be added?

Reply:
`lnbits core` should not know about a particular extension.
Suggestion: make the `Export` button a `<q-btn-dropdown>` with two options:
  - Basic Export
  - Detailed Export 

It defaults to `Basic Export`. Mock UI:

![image](https://github.com/lnbits/lnbits/assets/2951406/e3d39a8e-6cdc-43ac-ba72-e7c562e456f5)

Reply:
so but then, the detailed is not only for tpos, but it just packs all the info in the 'extra', is that it? if there's something there? this hack is just to include the `details` property that only exists, so far, in tpos

Reply:
I'm with Vlad, lets get this merged, and make something more sophisticated later.